access_test.go

```diff
@@ -272,6 +272,93 @@ func TestExtraScopes(t *testing.T) {
 
 }
 
+func TestPlatformToken(t *testing.T) {
+	sconfig := NewServerConfig()
+	sconfig.AllowedAccessTypes = AllowedAccessType{PLATFORM}
+	server := NewServer(sconfig, NewTestingStorage())
+	server.AccessTokenGen = &TestingAccessTokenGen{}
+	resp := server.NewResponse()
+
+	req, err := http.NewRequest("POST", "http://localhost:14000/appauth", nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	req.SetBasicAuth("1234", "aabbccdd")
+
+	req.Form = make(url.Values)
+	req.Form.Set("grant_type", string(PLATFORM))
+	req.Form.Set("client_id", "1234")
+	req.PostForm = make(url.Values)
+
+	if ar := server.HandleAccessRequest(resp, req); ar != nil {
+		ar.Authorized=true
+		server.FinishAccessRequest(resp, req, ar)
+	}
+
+	if resp.IsError && resp.InternalError != nil {
```

Comment:
is it enough just to assert the InternalError of resp? 
the error flag IsError is asserted on the next step.

Reply:
going to update it, thank you

Reply:
updated